README.md

```diff
@@ -76,7 +76,7 @@ Current available templates include:
 
 - [netlify] - Netlify CMS template using preact.
 
-- [typescript] - Default template implemented in TypeScript
+- [typescript] - Default template implemented in TypeScript, does not support `async!` import syntax.
```

Comment:
I wouldn't say that, the lack of `async!` support doesn't have anything to do with the template, just TS.

Reply:
I added a clarification that it's a TypeScript limitation rather than a template limitation.